sphinx_click/ext.py

```diff
@@ -291,13 +292,28 @@ def _format_command(ctx, show_nested, commands=None):
         yield ''
 
 
+def nested(argument):
+    values = "full", "short", "none"
+    if not argument:
+        return None
+
+    if argument not in values:
+        raise ValueError(
+            'must supply an argument; choose from %s'
+            % directives.format_values(values)
```

Comment:
Not sure what this API does. Assume it's equivalent to

    ', '.join(str(x) for x in values)

?

Reply:
I assume so; not sure if any special formatting is done. It's what built-in alternatives used for this scenario.